R/cv_estimate_rt.R

```diff
@@ -49,15 +49,15 @@ cv_estimate_rt <- function(
   if (nfold == 1) cli_abort("nfold must be greater than 1")
 
   ## Run program one time to create lambda
-  full_data_fit <- estimate_rt(
+  full_fit <- estimate_rt(
     observed_counts = observed_counts,
     korder = korder,
     x = xin,
     lambda = lambda,
     maxiter = maxiter,
```

Comment:
I run an example, and found `dist_gamma` had not been successfully passed into `estimate_rt()` from `cv_estimate_rt()`. Technically, it should be passed into `estimate_rt()` through the ellipsis, but it seems like it is not. I tried to include the argument `dist_gamma = dist_gamma` here and it fixed the problem. Not sure why the ellipse does not work here.